test/release/examples/primers/learnChapelInYMinutes.chpl

```diff
@@ -850,19 +861,24 @@ class GenericClass {
   var classDomain: domain(1);
   var classArray: [classDomain] classType;
 
-// Explicit constructor.
-  proc GenericClass(type classType, elements : int) {
-    this.classDomain = {1..#elements};
+// Explicit initializer.
+  proc init(type classType, elements : int) {
+    this.classType = classType;
+    this.classDomain = {1..elements};
+    // all generic and const fields must be initialized in "phase 1" prior
+    // to a call to the superclass initializer.
+    super.init(); 
   }
 
-// Copy constructor.
-// Note: We still have to put the type as an argument, but we can
-// default to the type of the other object using the query (``?``) operator.
-// Further, we can take advantage of this to allow our copy constructor
-// to copy classes of different types and cast on the fly.
-  proc GenericClass(other : GenericClass(?otherType),
-                     type classType = otherType) {
+// Copy-style initializer.
+// Note: We include a type argument whose default is the type of the first
+// argument.  This lets our initializer copy classes of different
+// types and cast on the fly.
+  proc init(other : GenericClass(?),
+            type classType = other.classType) {
```

Comment:
It doesn't really matter to me; I know what a *copy initializer* is for a record but I don't think we've any agreement on what the term is for the corresponding operation on a class. (and oddly enough that section of CHIP 10 seems to assume it's talking about a record without saying so).
However I think for the purposes of learnChapelInYMinutes I'd advocate just calling it an "initializer for performing a copy", "initializer for cloning", or some other wording that avoids the exact term "copy initializer". I don't really have a problem with "copy-style initializer" but I was responding to Lydia's comment.